release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -210,6 +210,10 @@ For more information, see xref:../installing/installing_with_agent_based_install
 You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the `install-config.yaml` file must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
 For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#understanding-disconnected-installation-mirroring[Understanding disconnected installation mirroring].
 
+[id="ocp-4-12-graph-true-field-disconnected"]
+==== Explanation of field to build and push graph-data
+When creating the image set configuration, you can add the `graph: true` field to build and push the graph-data image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS) applications. The `graph: true` field also generates the `UpdateService` CR manifest. The `oc` command-line interface (CLI) can use the `UpdateService` CR manifest to create OSUS applications.
```

Comment:
We should replace `create OpenShift Update Service (OSUS) applications` with `create OpenShift Update Service (OSUS)`. 

Replace `CR` with `custom resource` and `OSUS applications` OSUS`.